mesonbuild/utils/universal.py

```diff
@@ -1361,34 +1365,43 @@ def do_conf_file(src: str, dst: str, confdata: 'ConfigurationData',
 
 '''
 
-def dump_conf_header(ofilename: str, cdata: 'ConfigurationData', output_format: T.Literal['c', 'nasm']) -> None:
-    if output_format == 'c':
-        prelude = CONF_C_PRELUDE
-        prefix = '#'
-    else:
+def _dump_c_header(ofile: TextIOWrapper, output_format: str, cdata: ConfigurationData) -> None:
```

Comment:
Apparently I missed mentioning this before... We usually use T.TextIO here, which is also shorter:
```diff
-TextIOWrapper
+T.TextIO
```